reana_workflow_controller/rest/utils.py

```diff
@@ -345,18 +346,19 @@ def remove_workflow_workspace(path):
     :param path: Relative path to workspace directory.
     :return: None.
     """
-    reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
-    if reana_fs.exists(path):
+    # reana_fs = fs.open_fs(app.config["SHARED_VOLUME_PATH"])
+    if os.path.isdir(path):
         reana_fs.removetree(path)
 
 
 def mv_files(source, target, workflow):
     """Move files within workspace."""
-    absolute_workspace_path = os.path.join(
-        app.config["SHARED_VOLUME_PATH"], workflow.workspace_path
-    )
+    absolute_workspace_path = workflow.workspace_path
+    # os.path.join(
+    # current_app.config["SHARED_VOLUME_PATH"], workflow.workspace_path
+    # )
     absolute_source_path = os.path.join(
-        app.config["SHARED_VOLUME_PATH"], absolute_workspace_path, source
+        SHARED_VOLUME_PATH, absolute_workspace_path, source
```

Comment:
I think we don't need shared path to be mounted here?
isn't `absolute_source_path = os.path.join(workflow.workspace_path, source)` enough?

Reply:
Yes, the shared path is not needed in this case.